### ISSUE TITLE
Blocked subdomain breaks LG WebOS Appstore

### ISSUE THREAD

**Issue by cgprats** [hagezi/dns-blocklists#995]
### Which AdBlocker/DNS cloud service do you use?

NextDNS, ControlD, Other

### Which blocklist(s) do you use?

Multi NORMAL, Multi PRO

### Which domain(s) should be unblocked?

aic-gfts.lge.com

### Why should the domain(s) be unblocked?

This domain breaks the LG apps store application installation and updates. Update checks do work with the domain blocked, but the actual installation of the updates do not. Certain applications require you to be on the latest version (such as Netflix), meaning those applications are unusable well.

I have manually whitelisted this subdomain and LG apps began to work.

**Comment by hagezi** [comment]
Thanks, will be removed on next full update.

FYI: @markangus2b
https://github.com/hagezi/dns-blocklists/issues/309

**Comment by hagezi** [comment]
@cgprats fixed.

**Comment by markangus2b** [comment]
After adding `aic-gfts.lge.com` to my denylist on NextDNS it doesn't seem to break the LG content store. I was able to successfully install multiple apps. I even tried turning on the Block Page, Cache Boost, and CNAME Flattening in my NextDNS configuration, still no issues. I even fully rebooted my TV, still working.

My TV is from 2017 so it might be on a different version of WebOS that now requires this domain to function properly.
Model = OLED55C7P-U
Software Version = 06.10.30
webOS TV Version = 3.9.2-6270908

@cgprats If you don't mind, and maybe you've already done this. Try adding the domain to the denylist now, fully reboot your TV and see if it's still broken to confirm this domain is the culprit.

**Comment by cgprats** [comment]
> After adding `aic-gfts.lge.com` to my denylist on NextDNS it doesn't seem to break the LG content store. I was able to successfully install multiple apps. I even tried turning on the Block Page, Cache Boost, and CNAME Flattening in my NextDNS configuration, still no issues. I even fully rebooted my TV, still working.
> 
> My TV is from 2017 so it might be on a different version of WebOS that now requires this domain to function properly. Model = OLED55C7P-U Software Version = 06.10.30 webOS TV Version = 3.9.2-6270908
> 
> @cgprats If you don't mind, and maybe you've already done this. Try adding the domain to the denylist now, fully reboot your TV and see if it's still broken to confirm this domain is the culprit.

I have added it to my denylist on nextdns and can confirm that the issue is still present.

This is on a 50UP7670PUC on webOS6.0 build 03.34.65.

I am force redirecting all DNS to nextdns via my firewall. The TV does attempt to use Google DNS instead of NextDNS. Disabling redirect also fixes the issue, but allows the tv to completely bypass blocking.

**Comment by markangus2b** [comment]
> I am force redirecting all DNS to nextdns via my firewall. The TV does attempt to use Google DNS instead of NextDNS. Disabling redirect also fixes the issue, but allows the tv to completely bypass blocking.

Thank you for doing that. I was not aware that LG TVs fallback to Google DNS. I am currently using an Edgerouter X. Looks like I am going to be reconfiguring some firewall rules.

**Comment by hagezi** [comment]
@cgprats @markangus2b 

Thank you for the support. If DoH is used as bypass, then it is sufficient to switch on the bypass function in NextDNS or to use the DoH list. However, if native DNS (8.8.8.8 and co) or DoT is used, then it only helps to block the DNS and DoT ports in the firewall or router. The FritzBox can do this, for example, via the "parental control".

The bypass is now common practice in many apps and on some devices to bypass blocking.
